softlearning/samplers/utils.py

```diff
@@ -23,11 +23,13 @@ def get_sampler_from_variant(variant, *args, **kwargs):
     sampler_params = variant['sampler_params']
     sampler_type = sampler_params['type']
 
+    z_dim = sampler_params['z_dim']
+    z_type = sampler_params['z_type']
     sampler_args = sampler_params.get('args', ())
     sampler_kwargs = sampler_params.get('kwargs', {}).copy()
 
     sampler = SAMPLERS[sampler_type](
-        *sampler_args, *args, **sampler_kwargs, **kwargs)
+        z_dim = z_dim, z_type = z_type, *sampler_args, *args, **sampler_kwargs, **kwargs)
```

Comment:
pls remove whitespaces around argument assignments (I think that's convention) otherwise gets hard to read
i.e. z_dim=z_dim, z_type=z_type